File: 151004/Ermolovich/DC_Lab3/publisher/DC_Lab3/DC_Lab1/DTO/StoryRequestTo.cs
using DC_Lab1.DTO.Interface;
using DC_Lab1.Models;

namespace DC_Lab1.DTO
{
    public class StoryRequestTo(int Id, int? authorId, string? Title, string? Content) : IRequestTo
    {
        public int Id { get; set; } = Id;

        public int? authorId { get; set; } = authorId;

        public string? Title { get; set; } = Title;

        public string? Content { get; set; } = Content;


    }
}

File: 151001/Manchik/publisher/src/main/java/application/dto/StoryRequestTo.java
package application.dto;

import application.entites.utils.JsonCDeserializer;
import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
import jakarta.validation.constraints.NotBlank;
import jakarta.validation.constraints.Size;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.io.Serializable;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class StoryRequestTo implements Serializable {
    private Long id;
    @NotBlank
    @Size(min =  2, max = 64)
    private String title;
    @NotBlank
    @Size(min =  4, max = 2048)
    @JsonDeserialize(using = JsonCDeserializer.class)
    private String content;
    private Long authorId;
    private Long markerId;
}
